### PR TITLE
Post a message to #release upon completion

### PR DESCRIPTION
**Pull request by alexcb**
Signed-off-by: Alex Couture-Beil <alex@earthly.dev>

### CODE REVIEW

File: release/Earthfile [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+RUN apk add --update --no-cache \
+    curl
+
 release:
     ARG RELEASE_TAG
     RUN test -n "$RELEASE_TAG"
     BUILD --build-arg RELEASE_TAG="$RELEASE_TAG" +release-dockerhub
     BUILD --build-arg RELEASE_TAG="$RELEASE_TAG" +release-github
+    RUN --push --secret SLACK_WEBHOOK_URL=+secrets/earthly-technologies/slack/release-webhook \
+        curl -X POST -H 'Content-type: application/json' --data '{"text":"Successfully released '$RELEASE_TAG'"}' $SLACK_WEBHOOK_URL
```
Comment (vladaionescu) [inline release/Earthfile:13]: Minor: put the version in back ticks.

```suggestion
        curl -X POST -H 'Content-type: application/json' --data '{"text":"Successfully released `'$RELEASE_TAG'`"}' $SLACK_WEBHOOK_URL
```

**Review by vladaionescu — COMMENTED** [review]
Yay! Could also add the Homebrew PR link during homebrew release.

**Comment by alexcb** [comment]
to get the homebrew PR, do we need to parse the output of `hub pull-request ...` under https://github.com/earthly/earthly/blob/release-notifier/release/Earthfile#L156-L170

I'm trying to figure out how to test this without actually doing a release :thinking:

**Comment by vladaionescu** [comment]
Oh the output is just the URL apparently: 

```
./release+release-homebrew | --> RUN --push version=${RELEASE_TAG#v} ;echo version=$version ;hub pull-request --no-edit -h "$GIT_USERNAME":"release-$RELEASE_TAG" -b "$REPO_OWNER:master" -m "earthly $version" -m '-------------' -m '#### Debug data' -m 'PR generated by the [Earthly build](https://github.com/earthly/earthly/blob/main/release/Earthfile) (target +release-homebrew)' -m '* `RELEASE_TAG='"$RELEASE_TAG"'`' -m '* `GIT_USERNAME='"$GIT_USERNAME"'`' -m '* `NEW_URL='"$NEW_URL"'`' -m '* `NEW_SHA256='"$(cat /params/downloadsha256)"'`' -m '* `TAGS='"$(cat /params/tags)"'`' -m '* `LDFLAGS='"$(cat /params/ldflags)"'`' -m '* `LDFLAGS1='"$(cat /params/ldflags1)"'`' -m '* `LDFLAGS2='"$(cat /params/ldflags2)"'`'
./release+release-homebrew | version=0.3.19
./release+release-homebrew | https://github.com/Homebrew/homebrew-core/pull/66255
```

So no need to parse. It looks good.

**Review by vladaionescu — APPROVED** [review]

**Pull request merged**